cmd/juju/cloud/list_test.go

```diff
@@ -56,7 +56,7 @@ clouds:
 	// Just check a snippet of the output to make sure it looks ok.
 	// local clouds are last.
 	// homestack should abut localhost and hence come last in the output.
-	c.Assert(out, jc.Contains, `Hypervisorhomestack          1  london           openstack   Openstack Cloud`)
+	c.Assert(out, jc.Contains, `Hypervisorhomestack          1  london         openstack   Openstack Cloud`)
```

Comment:
really? how come?

Reply:
It's tabular output and the width is affected by changing the oracle cloud text

Reply:
I'm not sure how this stuff gets used but shouldn't it be `OpenStack cloud`?

Reply:
All of our cloud descriptions use camel case.